### DISCUSSION TITLE
Song title and artist name should be clickable on Song Select menu.

### DISCUSSION THREAD

**Discussion by FazlyMR** [ppy/osu#22332] [Ideas] [answered]
To start, I'm aware of the currently ongoing song select menu redesign (#22204), but in the Song Select menu, if the mapper's profile name is already clickable and leads to their profile if clicked, both song title and artist name should also be clickable that leads to the Browse menu with either one clicked by the user already inputted into the search bar for them or the Song Select search bar if a user wants to search other beatmaps in their device from the same artist.

(I'm not sure on how should clicking to search on Browse or in the Song Select menu should be differentiated when a user clicks either of them, but there should somehow be a way.)

Having to right-click a map, selecting "Details", then either song title or artist name to begin a search I think is unnecessary.

Mapper's profile is clickable, and leads to their profile if clicked.

[seg1.webm](https://user-images.githubusercontent.com/69378798/213894272-cb9e8e48-23e0-4920-a596-4112cf1364b1.webm)

Song title and artist name are not clickable unless one right-clicks a song/beatmap on the right side, then select "Details" first.

[seg2.webm](https://user-images.githubusercontent.com/69378798/213894307-0c81645f-dfdd-4633-97e9-dc3e17686a95.webm)

[seg3.webm](https://user-images.githubusercontent.com/69378798/213894312-819543dc-8d76-48a3-b09b-b55f2010feba.webm)

**Comment by Joehuu** [comment, marked as answer]
I have a local change pending that addresses this but is waiting on the redesign in https://github.com/ppy/osu/pull/22116.

Also, the behavior will be different by default (i.e. it'll fill out the song select search textbox instead like tags and source are) unless we want both behaviors accessible by right-clicking the text.

**Reply by FazlyMR** [reply]
Ok then, thanks for the answer.

But, can you please elaborate a little bit on "**instead like tags and source are**", I'm a little confused by that sentence?

**Reply by Joehuu** [reply]
I'm talking about this:

![osu!_TTD8qM5fUL](https://user-images.githubusercontent.com/35318437/213895589-c140de8e-12e7-4f39-809f-a4e6d84cc096.gif)

**Reply by Joehuu** [reply]
Have implemented in new design: https://github.com/ppy/osu/pull/24733
